data/application.css

```diff
@@ -15,6 +15,10 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+.terminal {
```

Comment:
Since this is handled by the GTK theme already, it should probably be updated there: https://github.com/pop-os/gtk-theme/blob/4ffc89a4869936db0796d949774000612eea7734/src/gtk-3.0/3.22/sass/widgets/_misc.scss#L46

For example, the elementary stylesheet already styles this correctly, and we don't want to override that here https://github.com/elementary/stylesheet/blob/eaa32160ed2801b489898e2c640c5ef81f5ffafd/gtk-3.22/granite-widgets.css#L657